README.md

```diff
@@ -2446,7 +2446,7 @@ console.log(`SQLite 3 version: ${sqlite3_libversion()}`);
 
 #### Low-overhead FFI
 
-3ns to go from JavaScript <> native code with `bun:ffi` (on my machine, an M1X)
+3ns to go from JavaScript <> native code with `bun:ffi` (on my machine, an M1 Pro with 64GB of RAM)
```

Comment:
The part updating the CPU is nice.

But the numbers are good to include

Reply:
Not sure what you're referring to, I didn't remove the numbers unless I messed up my edit somehow?

Reply:
ah, I misread the diff. But viewing the page shows its the same

<img width="288" alt="image" src="https://user-images.githubusercontent.com/709451/177682016-05ec992a-3526-4dc2-88e0-9de0c9e2bfbc.png">